manifests/tekton/tasks/base/cosa-build-baseos.yaml

```diff
@@ -21,9 +31,18 @@ spec:
 
         cd /srv/coreos
 
+        if [[ -z "$(params.endpoint-url)" || -z "$(params.aws-config-file)" || -z "$(params.bucket-name)" ]]; then
```

Comment:
This should allow the pipeline runs on kind to still pass, ignoring the buildfetch